src/main/scala/com/tresata/spark/scalding/SparkJob.scala

```diff
@@ -18,4 +25,44 @@ abstract class SparkJob(args: Args) extends Job(args) {
   implicit lazy val sc: SparkContext = newSc(ConfigFactory.load, name, tmpJars)
 
   override def run : Boolean
+
+  /**
+   * Overrides the Scalding Job config method in order not to use the
+   * default serializations, but to pass different ones instead.
+   * 
+   * @return
+   */
+  override def config: Map[AnyRef, AnyRef] = {
+    // this will overwrite the io.serializations option to what
+    // we need it to be.
+    super.config ++ setSerialization(ioSerializations).toMap
+  }
+
+  /**
+   * Hadoop and Cascading serialization needs to be first, and the Kryo serialization
+   * needs to be last and this method handles this for you:
+   * hadoop, cascading, [userHadoop,] kyro
+   * is the order.
+   *
+   * Kryo uses a specific class here.
+   *
+   * @param userHadoop
+   * @return
+   */
+  private[scalding] def setSerialization(userHadoop: Seq[Class[_ <: HSerialization[_]]] = Nil): Config = {
```

Comment:
does it need to be private[scalding]? can it be private, or perhaps protected in case someone wants to call it in a Job that extends SparkJob to add some other serializations?